### ISSUE TITLE
ClinSV v1.1

### ISSUE THREAD

**Issue by J-Bradlee** [KCCG/ClinSV#27]
### Description

Implementation of a few features that will make ClinSV more robust and efficient. 

### Features to be implemented:

- [ ] Currently unable to switch between hs37d5 and grch38 on the fly, must use version 0.9 for  hs37d5 and version 1.0 for grch38. Ideally we want to have one version of ClinSV to be able to easily switch between the two.
- [ ] Currently there is no support for hg19 style chromosome names for v37 or v38 reference genomes. We should provide support for this.
- [ ] Not all resources available to ClinSV are being used. It is currently hard coded as some preset values. We want to be able to have ClinSV dynamically use resources available to speed up the computation.

### Bug fixes
- [ ] The igv.xml file outputted when using the grch38 ref genome is incorrect as it is based on grch37. Provide the appropriate igv.xml file dependent on ref genome.

**Comment by drmjc** [comment]
Thanks @J-Bradlee, this is a good summary of the scope of v1.1.0

If the Docker/clinsv_scripts.tar.gz file includes a copy of the files in the scripts/* folder, can you drop the need for the clinsv_scripts.tar.gz file in this release too?

**Comment by J-Bradlee** [comment]
Good point @drmjc I'll change up the docker file to pull directly from git, so we no longer need to have the zip files.

**Comment by EricDeveaud** [comment]
seems to me that https://github.com/KCCG/ClinSV/commit/18e7603293947ac49f4e82233e40202a92aa19b4

should be applied to 1.1-dev branch to
